content/en/blog/call-for-blogs.md

```diff
@@ -0,0 +1,42 @@
+# Call for blog contributions
+
+This is a constantly updated list of the blogs that the sig-docs is interested in.
+
+This will list the blogs along with abstract that can be furthre used for gathering information.
```

Comment:
This line should be inviting folks and asking them to reach out to us if they have any questions (on Slack) before they submit a PR to k/website. WDYT?

Reply:
From the issue, I'm thinking the introductory line will focus on mentioning a detail about some important blogs that most of the users are looking forward to. For eg. 

Folks would be interested to know more about Kubernetes:v1.26 release features - why it's an electrifying release?

However, the last line of the introduction can mention reaching out to slack for queries and references on how to submit blogs to Kubernetes. WDYT @divya-mohan0209 about this?

Reply:
Could you PR the changes in and then we'll see if it fits in? I am not able to understand what you're trying to convey.